notif.go

```diff
@@ -109,6 +109,16 @@ func (nn *netNotifiee) Disconnected(n network.Network, v network.Conn) {
 	}
 
 	dht.routingTable.Remove(p)
+	if dht.routingTable.Size() < minRTRefreshThreshold {
+		// TODO: Actively bootstrap. For now, just try to add the currently connected peers.
+		for _, p := range dht.host.Network().Peers() {
```

Comment:
@Stebalien  

1) We add a peer to the RT when we connect to it in the Dht Connected notification handler...except for when it does not meet the latency requirements 

```
if rt.metrics.LatencyEWMA(p) > rt.maxLatency {
		// Connection doesnt meet requirements, skip!
		return "", ErrPeerRejectedHighLatency
	}
```

Am I to understand that this code will ONLY add these high latency peers to the RT because if we are connected to a peer with an acceptable latency, it should already be in the RT.

Reply:
If so,

a better logic IMO would be:

1. If rt.Size() <  minRTRefreshThreshold, trigger a bootstrap by sending a req to the channel.
2. If rt.Size() == 0, add the high latency peers as a last resort(temporarily, till we land #387 )